nestkernel/conn_builder.h

```diff
@@ -291,11 +293,168 @@ class ConnBuilder
   void reset_delays_();
 };
 
-class OneToOneBuilder : public ConnBuilder
+// not expected to be subclassed further
+class ThirdInBuilder : public BipartiteConnBuilder
```

Comment:
Could you add a docstring to ThirdInBuilder and OutBuilder? I don't understand what their reponsibilities are.

Reply:
Done, I hope they are sufficient.